types/src/task_state.rs

```diff
@@ -127,6 +127,7 @@ impl Task<Create> {
         if !function.public {
             participants.insert(function.owner.clone());
         }
+        log::info!("participants {:?}", &participants);
```

Comment:
Why do we need this log? Also, participants should be a sensitive data, this is a info log.

Reply:
I have removed this log.